warrior/WarriorCore/Classes/hybrid_driver_class.py

```diff
@@ -213,6 +214,36 @@ def _execute_step_list(self):
             if self.stop_after_current_step:
                 break
 
+            # Decide whether or not to execute keyword
+            # First decide if this step should be executed in this iteration
+            if not goto_stepnum or goto_stepnum == str(step_num):
+                # get Exectype information
+                run_current_step, trigger_action = exec_type_driver.main(step)
+                if not run_current_step:
+
+                    if trigger_action.upper() in ['ABORT', 'ABORT_AS_ERROR']:
```

Comment:
if the action is abort, stop_after_current_execution and stop_after_current_step should be set only if the iteration type is not standard.. this is because for standard iterative behavior when the action is abort we only abort the current iteration and move on to the next iteration.

Reply:
Addressed this comment in the 8b18946d82f5bf30e28e989dcac1c5efb160fe89